src/stats.jl

```diff
@@ -228,8 +227,6 @@ end
         append_chains= true,
         method::AbstractESSMethod = ESSMethod(),
         maxlag = 250,
-        etype = :bm,
-        kwargs...
```

Comment:
It seems `kwargs...` are still supported and should not be removed? Is `etype=:bm` still the default?